psychopy/visual/target.py

```diff
@@ -138,17 +130,23 @@ def lineWidth(self, value):
 
     @property
     def radius(self):
+        """
+        TODO: Deprecate use of radius in favor of using .size
+        :return:
+        """
         return self.outerRadius
 
     @radius.setter
     def radius(self, value):
         # Set outer radius
         self.outerRadius = value
-        # Set inner radius to maintain scale
```

Comment:
So rather than maintaining the ratio between inner an outer, setting radius sets the outer radius and setting innerRadius sets the inner? I guess this is fine, but what's the reason for the change? It does mean that if you wanted to expand the whole thing (maintaining ratio) it's harder to do

Reply:
The purpose of the my original target stim was to never change the inner circle radius when the target expands / contracts, only the outer circle radius. This was very intentional. So this is more in line with the original intent of the stim and keeps it operating as before.

| I guess this is fine, but what's the reason for the change?

Switching to use `size` makes the calibration code for different stim types much more sharable / the same as possible across stim types.

| It does mean that if you wanted to expand the whole thing (maintaining ratio) it's harder to do

This was not possible before during calibration / validation either. Now if a user wants that type of behaviour they can do it using an image or any custom stim they make.

Reply:
Cool, all seems reasonable :) In that case consider my review approved